### PR TITLE
Added link to structure.md

### PR DESCRIPTION
**Pull request by adam-thorpe**
# Checklist

- [x] You added tests to cover the change
- [x] `mvn clean install` build and test completes
- [x] You changed or added to the documentation

Closes: #186 
Signed-off-by: Adam Thorpe <adam.thorpe@ibm.com>

### CODE REVIEW

File: adoptopenjdk-api-v3-frontend/src/main/resources/META-INF/resources/index.html [hunk 1]
```diff
@@ -27,6 +27,7 @@ <h2>Overview</h2>
 
     <h2>Usage</h2>
     <p>View usage at <a href="swagger-ui">swagger-ui docs</a></p>
+    For more information, including example queries, please look at <a rel="noopener noreferrer" href="https://github.com/AdoptOpenJDK/openjdk-api-v3/blob/master/docs/STRUCTURE.md">STRUCTURE.md</a>
```
Comment (johnoliver) [inline adoptopenjdk-api-v3-frontend/src/main/resources/META-INF/resources/index.html:30]: should probably put that in a <p>

Comment (sxa) [reply adoptopenjdk-api-v3-frontend/src/main/resources/META-INF/resources/index.html:30]: @johnoliver what was the end of that sentence? :-)

Comment (johnoliver) [reply adoptopenjdk-api-v3-frontend/src/main/resources/META-INF/resources/index.html:30]: turns out github escapes `<p>`

Comment (adam-thorpe) [reply adoptopenjdk-api-v3-frontend/src/main/resources/META-INF/resources/index.html:30]: Oh yeah I'm dumb I dunno how I missed those

**Pull request merged**